cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -0,0 +1,788 @@
+    Scenario: UserGrants.Grant R*2.2.2.1 - Positive Scenario: Validate Capabilities with denied (pinChallenge) and lifespan once
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        # Case-1
+        And Framework registers 'pinChallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'get postalcode'
+        Then 'Firebolt' platform responds with 'expected value for localization postalcode'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for postalcode'
+        Then 'Firebolt' platform responds with 'null for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+        # Case-2
+        When User set response for 'set pinchallenge wrong pin'
+        And User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'expect error for postalcode'
+        Then 'Firebolt' platform responds with 'invalid request for postalcode'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for postalcode'
+        Then 'Firebolt' platform responds with 'null for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+
+    # Testing with grantPolicy having one option with one step(here step with acknowledge)
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities R.2.1.1 - Positive Scenario: Validate Capability Grant access with one step option(only acknowledgeChallenge)
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        And Framework registers 'ackchallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'get localization countrycode'
+        Then 'Firebolt' platform responds with 'expected localization countrycode'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for localization countrycode'
+        Then 'Firebolt' platform responds with 'true for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+
+    # Testing with grantPolicy having with scope device and grant access, so all the apps in that device may have the access
+    # provide access using any 3rd party and check in another 3rd party
```

Comment:
No need to port this test case. Instead add a comment in https://ccp.sys.comcast.net/browse/FIRECERT-2194 to include these TC

Reply:
This would require using secondary 3rd party app and hence requires test steps to be modified accordingly

Reply:
2 apps testcases ported in 2194, Added comment in 2195 to port multiple device TC